FRW/Modules/Profile/ProfileSecure/ProfileSecureViewViewModel.swift

```diff
@@ -21,11 +21,12 @@ class ProfileSecureViewModel: ObservableObject {
         }
 
         if !isEnabled {
+            EventTrack.General.security(type: .none)
             SecurityManager.shared.disableBionic()
             isBionicEnabled = false
             return
         }
-
+        EventTrack.General.security(type: .bionic)
```

Comment:
It should be `.biometric` instead of `.bionic`

Reply:
track event name is `biometric`, `.bionic` is enum on SecurityManager.